lib/availability.ts

```diff
@@ -45,3 +52,61 @@ export function getAvailabilityFromSchedule(schedule: Schedule): Availability[]
     return availability;
   }, [] as Availability[]);
 }
+
+/**
+ * PostgreSQL stores the date relatively
+ */
+export function getWorkingHours(
+  timeZone: string,
+  availability: Pick<Availability, "days" | "startTime" | "endTime">[]
+) {
```

Comment:
This is the sort of places where I'd actually want unit tests :D

Reply:
yep! I've been intending to write one but there is also a lot of "prototyping" in this "Does this thing that makes sense in my mind actually work?". As a result the tests are a bit secondary atm; but I think there must be one to cover this function before this is merged (even though it's an important merge)